pegasus/sites.v3/code.org/views/promote_state.haml

```diff
@@ -137,9 +137,6 @@
     font-weight: bold;
     text-align:center;
   }
-  .fact {
-    font-size:15px;
```

Comment:
teeny tiny nit: space between font-size and 15px

Reply:
Fortunately that is being removed :)